src/models/batch_rewards_schema.py

```diff
@@ -32,6 +32,7 @@ def from_pdf_to_dune_records(cls, rewards_df: DataFrame) -> list[dict[str, Any]]
                     "execution_cost": int(row["execution_cost"]),
                     "surplus": int(row["surplus"]),
                     "fee": int(row["fee"]),
+                    "protocol_fee": int(row["protocol_fee"]),
```

Comment:
Should we maybe add this at the very end? Not sure we use indices at any point (I suspect not), but just speculating here...

Reply:
The ordering in this data structure should not matter sice it uses a python dict, which does not have an order.

I think the ordering is a bit inconsistent between different repos. We should probably stick with whatever we find best for readability. We could put it at the end or group with surplus, fees, etc.